client/assets/stylesheets/_wp-components-overrides.scss

```diff
@@ -2,3 +2,77 @@
 .components-form-toggle.is-checked .components-form-toggle__track {
 	background-color: var( --color-primary );
 }
+
+/* @wordpress/components Button overrides */
+.components-button {
+	color: var( --color-neutral-70 );
+
+	&:hover:not( :disabled ) {
```

Comment:
Since we have the same `color` rule here, perhaps we could avoid redefining it by doing:

```
&,
&:hover:not( :disabled ) {
	color: var( --color-neutral-70 );
}
```

Reply:
Aha! Always a good Sass trick around the corner 😉